integrations/deploy.yaml

```diff
@@ -0,0 +1,48 @@
+- &linux
+  id: deploy-linux-generic
+  meta: &linux_meta
+    name: Linux
+    link: ''
+    categories:
+      - deploy.operating-systems
+    icon_filename: ''
+  keywords:
+    - linux
+  description: 'Run the following command on your node to install and claim Netdata:"r'
+  methods:
+    - method: wget
+      commands:
+        - channel: nightly
+          command: >
+            wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --nightly-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+        - channel: stable
+          command: >
+            wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --stable-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+    - method: curl
+      commands:
+        - channel: nightly
+          command: >
+            curl https://my-netdata.io/kickstart.sh > /tmp/netdata-kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --nightly-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+        - channel: stable
+          command: >
+            curl https://my-netdata.io/kickstart.sh > /tmp/netdata-kickstart.sh && sh /tmp/netdata-kickstart.sh
+            --stable-channel --claim-token {% claim_token %} --claim-url {% claim_url %}
+  additional_info: &ref_containers >
+    Did you know you can also deploy Netdata on your OS using Kubernetes or Docker?
```

Comment:
We need to link Kubernetes and Docker sections to the relevant Deploy sections

Reply:
I’m not certain what format is going to be needed for that, so I just copied the text for now.

We’re going to need to ask @kapantzak what’s needed for this.

Reply:
@Ferroin we should use a custom tag there.

I think both of the links (`Kubernetes`, `Docker`) are going to do the same thing (redirect user to the `Docker & Kubernetes` category) right?

So just use this:

```
Did you know you can also deploy Netdata on your OS using {% GoToCategory categoryId="deploy.docker-kubernetes" %}Kubernetes{% /GoToCategory %} or {% GoToCategory categoryId="deploy.docker-kubernetes" %}Docker{% /GoToCategory %}?
```

Reply:
@Ferroin a small change I mage to the above, better use lower case tag name:

```
GoToCategory --> goToCategory
```